contents/docs/tutorials/b2b.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Tracking Key B2B Product Metrics
+sidebar: Docs
+showTitle: true
+---
+
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 10 minutes ☕☕☕_</small>
+
+<br />
+
+<span class="larger-image">
+
+![B2B Tutorial Banner Image](../../images/tutorials/banners/b2b.png)
+
+</span>
+
+<br />
+
+While there is some overlap between the analytics processes of B2C and B2B businesses, there are significant differences in the business model that need to be accounted for when tracking key performance metrics.
```

Comment:
This is a bit of a non sentence. Could we make it a little punchier?

"If you're in B2B, it's very important to understand _group_ usage of your product, and there are plenty of good reasons for this... ie.", or something like that.